lumicks/pylake/channel.py

```diff
@@ -84,13 +84,19 @@ def _unpack_other(self, other):
     def __neg__(self):
         if isinstance(self._src, TimeTags):
             raise NotImplementedError("This operation is not supported for TimeTag data")
-        return Slice(self._src._with_data(-self.data))
+        return Slice(self._src._with_data(-self.data), calibration=self._calibration)
 
     def __add__(self, other):
-        return Slice(self._src._with_data(self.data + self._unpack_other(other)))
+        return Slice(
+            self._src._with_data(self.data + self._unpack_other(other)),
+            calibration=self._calibration if np.isscalar(other) else None,
+        )
```

Comment:
I don't see `__radd__(self, other)` though it's also supposed to preserve calibration. Intentional?

Reply:
https://github.com/lumicks/pylake/blob/1b51ad75519f81e8bfc532ea8f6f56ca5cb2abc6/lumicks/pylake/channel.py#L121 :)

Reply:
Ah, the ol' commutative property :ok_hand: